### ISSUE TITLE
4.0.4 build error on Ember 3.28.8

### ISSUE THREAD

**Issue by kategengler** [cibernox/ember-basic-dropdown#656]
```
Build Error (broccoli-persistent-filter:Babel > [Babel: @embroider/macros]) in @embroider/macros/runtime.js

/Users/katie/dev/codeallday/app-name/@embroider/macros/runtime.js: @babel/template placeholder "LOG_VIEW_LOOKUPS": Expected string substitution
```
This happens moving from 4.0.3 to 4.0.4 on an Ember 3.28.8 app. All ember-power-* addons are on latest versions.

**Comment by csprocket777** [comment]
I had a similar issue that was resolved by pinning `@embroider/macros@1.2.0` in my resolutions.

**Comment by ef4** [comment]
Yes, this is a bug in a quite old copy of `@embroider/macros`. It's generally safe to force upgrade it to `^1.0`.

**Comment by cibernox** [comment]
I updated embroider packages to ^1.2.0 and released 4.0.5. Hopefully that solves the issue.

**Comment by joshkg** [comment]
Note you may have other addons depending on older version of `@embroider/macros`-- for me it was `ember-changeset`, adding a `resolutions` in `package.json` and pinning to `1.3.0` seems to be working.
